### PR TITLE
Initial entry for mlab_diff slice.

### PR DESCRIPTION
**Pull request by functor**
While adding the new slice mlab_diff, it would make sense to add it to m-lab/operator so that it all stays consistent and well defined while making any relevant changes. This is a start.

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/operator/189)

### CODE REVIEW

File: plsync/slices.py [hunk 1]
```diff
@@ -142,6 +142,10 @@
                                             ipv6="all",
                                             rsync_modules=['utilization', 'switch']),
 
+    Slice(name="mlab_diff",       index=12, attrs=centos_slice_attrs+web100_enable_attr,
```
Comment (critzo) [inline plsync/slices.py]: @functor This slice won't need to have web100 stuff.  So this line can be trimmed a little. If you could amend with another commit, I think we're good.

**Comment by stephen-soltesz** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[plsync/slices.py, line 145 at r1](https://reviewable.io:443/reviews/m-lab/operator/189#-L47QiaZsBc3ikNXcUYy-r1-145:-L47b2BfYntsCuCAdJgY:b-7p0woo) ([raw file](https://github.com/m-lab/operator/blob/b7e8d12c143650e13b88ef0e43dce48a51d82637/plsync/slices.py#L145)):*
<details><summary><i>Previously, critzo wrote…</i></summary><blockquote>

@functor This slice won't need to have web100 stuff.  So this line can be trimmed a little. If you could amend with another commit, I think we're good.
</blockquote></details>

Sorry to drop in uninvited, just want to save a round trip -- slice indexes are from 0-11 https://github.com/m-lab/operator/blob/master/plsync/slices.py#L67

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/operator/189)*

File: plsync/slices.py (hunk 1 above)
Comment (critzo) [reply plsync/slices.py]: @functor I have another commit to make to operator, so I'm going to add your work to it, and close this PR.

**Comment by critzo** [comment]
Thanks @stephen-soltesz

**Pull request closed without merging**